### PR TITLE
Introduce transfer crystal white listing

### PR DESCRIPTION
**Pull request by ipdae**
- allow transfer crystal with set block index(6,200,000) with white list.

### CODE REVIEW

File: Lib9c/Action/TransferAsset.cs [hunk 1]
```diff
@@ -9,21 +9,32 @@
...
-    /// Updated at https://github.com/planetarium/lib9c/pull/957
+    /// Updated at https://github.com/planetarium/lib9c/pull/1718
     /// </summary>
     [Serializable]
     [ActionType("transfer_asset3")]
     public class TransferAsset : ActionBase, ISerializable, ITransferAsset, ITransferAssetV1
     {
         private const int MemoMaxLength = 80;
 
+        public const long WhiteListAvailableIndex = 6_200_000L;
```
Comment (moreal) [inline Lib9c/Action/TransferAsset.cs:28]: I think this interval is too close 🤔 I think it should be bigger as 300000 than the current block index (6087423)

Comment (longfin) [inline Lib9c/Action/TransferAsset.cs:28]:
```suggestion
        public const long CrystalTransferringRestrictionStartIndex = 6_200_000L;
```

if we feel `CrystalTransferringRestriction` is too verbose, leveraging an external NCIP number (if exists) would be good too. cc @jaeho0103

File: Lib9c/Action/TransferAsset.cs [hunk 2]
```diff
@@ -9,21 +9,32 @@
...
+
+        public static readonly IReadOnlyList<Address> WhiteList = new List<Address>
```
Comment (longfin) [inline Lib9c/Action/TransferAsset.cs:30]:
```suggestion
        public static readonly IReadOnlyList<Address> AllowedCrystalTransferers = new List<Address>
```

Comment (longfin) [reply Lib9c/Action/TransferAsset.cs:30]: Is there any external resource(e.g., NCIP, DAO proposal) to justify their addresses? @ipdae @jaeho0103

File: Lib9c/Action/TransferAsset.cs [hunk 3]
```diff
@@ -9,21 +9,34 @@
...
     [ActionType("transfer_asset3")]
     public class TransferAsset : ActionBase, ISerializable, ITransferAsset, ITransferAssetV1
     {
         private const int MemoMaxLength = 80;
 
+        // FIXME justify this policy.
+        public const long CrystalTransferringRestrictionStartIndex = 6_220_000L;
+
+        // FIXME justify this policy.
+        public static readonly IReadOnlyList<Address> AllowedCrystalTransferList = new List<Address>
```
Comment (longfin) [inline Lib9c/Action/TransferAsset.cs:32]:
```suggestion
        public static readonly ISet<Address> AllowedCrystalTransfers = new List<Address>
```

Sorry for the late & trivial checking. I think we don't need to keep the order. 😅

**Review by longfin — APPROVED** [review]

File: Lib9c/Action/TransferAssets.cs [hunk 4]
```diff
@@ -9,6 +9,7 @@
 using System.Linq;
 using System.Runtime.Serialization;
 using Lib9c.Abstractions;
+using Nekoyume.Helper;
```
Comment (moreal) [inline Lib9c/Action/TransferAssets.cs:12]: This addition seems unnecessary.

```suggestion
```

**Review by moreal — APPROVED** [review]

**Pull request merged**